src/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_FIREBASE_KEY: AIzaSyC9w0L8Eva7O3OOsz8qt7gxaK2BJLQhEwU
+REACT_APP_FIREBASE_DOMAIN:creative-prompts.firebaseapp.com
+REACT_APP_FIREBASE_DATABASE:https://creative-prompts.firebaseio.com
+REACT_APP_FIREBASE_PROJECT_ID:
+REACT_APP_FIREBASE_STORAGE_BUCKET:creative-prompts.appspot.com
+REACT_APP_FIREBASE_SENDER_ID:689694289212
```

Comment:
Not sure if this is an error, but github gives me some red circle at the end complaining theres no newline at the end of the file lol

Reply:
and in some of the other files.

Reply:
Fixed! And ooh also I looked this one up! LOL
The warning is because its good practice to include a new line after every file. It comes from the C standard because when the C preprocessor concats files together, it makes sure that two instructions don't pile up on the same line.
Normally this isn't a problem because text editors like sublime and atom automatically adds a new line for you at the end of every file. But for this I used VS Code which doesn't do that..... ahh smh Microsoft.......